### PR TITLE
backwards compatibility for unicode struct specs

### PR DESCRIPTION
**Pull request by blampe**
@breerly @prashantv @junchaowu 

Need to add a test.

I don't like the repeated `str` calls, might look into ensuring these specs remain bytes.

### CODE REVIEW

**Comment by breerly** [comment]
fixes #26

**Comment by breerly** [comment]
:shipit:

**Comment by junchaowu** [comment]
lgtm

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Releases
 0.3.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Backwards compatibility for Python 2.7.6 and earlier due to Python #19099.
```
Comment (breerly) [inline CHANGELOG.rst]: python 2.7.3*

**Comment by breerly** [comment]
ship it

File: tests/protocol/test_binary.py [hunk 2]
```diff
@@ -364,3 +365,35 @@ def test_unknown_type_id(typ, bytes):
     with pytest.raises(ThriftProtocolError) as exc_info:
         reader.read(typ)
     assert 'Unknown TType' in str(exc_info)
+
+
+@pytest.mark.parametrize('typ, bytes, value', [
+    (typ, 0x01, vi16(0)) for typ in TType
+], ids=reader_writer_ids)
+def test_backwards_compatibility_with_python_266(typ, bytes, value):
```
Comment (breerly) [inline tests/protocol/test_binary.py]: sweet

**Pull request merged**